src/_langs/en/updates/2015-08-28-using-web-app-manifest-to-set-solid-color-loading-screen.markdown

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
This looks like a duplicate of the content above. Please copy whatever edits you make.